shopfloor_packing_info/models/shopfloor_packing_info.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Camptocamp SA (https://www.camptocamp.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+from odoo import fields, models
+
+
+class ShopfloorPackingInfo(models.Model):
+    _name = "shopfloor.packing.info"
+    _description = "Shopfloor Packing Information"
+
+    name = fields.Char("Name", required=True)
+    text = fields.Text("Text", required=True)
+    active = fields.Boolean("Active", default=True)
```

Comment:
```suggestion
    name = fields.Char(required=True)
    text = fields.Text(required=True)
    active = fields.Boolean(default=True)
```

The string is generated from the field name already, better not to set it when it's the same